lineage/tests/test_StateDistribution.py

```diff
@@ -0,0 +1,30 @@
+""" Unit test file. """
+import unittest
+import numpy as np
+import scipy.stats as sp
+from ..StateDistribution import bernoulli_estimator, exponential_estimator, gamma_estimator
+
+
+class TestModel(unittest.TestCase):
+    """Here are the unit tests."""
+
+    def test_bernoulli_estimator(self):
+        """ blah """
+        bern_obs = sp.bernoulli.rvs(p=0.9, size=1000)  # bernoulli observations
+        self.assertTrue(0.88 <= bernoulli_estimator(bern_obs) <= 0.92)
+
+    def test_exponential_estimator(self):
+        """ blah """
+        exp_obs = sp.expon.rvs(scale=50, size=1000)  # exponential observations
+        self.assertTrue(45 <= exponential_estimator(exp_obs) <= 55)  # +/- 5 of beta
```

Comment:
how did you decided upon 5 and .02 for the bern as the up/down boundaries

Reply:
These were just the ones that caused the tests to fail the least.